attest/verifier/build.rs

```diff
@@ -86,12 +91,19 @@ fn main() {
         generate_sim_files(&data_path);
     }
 }
+
+const ROOT_ANCHOR_FILENAME: &str = "root_anchor.pem";
+const SIGNER_KEY_FILENAME: &str = "signer.key";
+const CHAIN_FILENAME: &str = "chain.pem";
+
+const GENERATED_FILENAMES: &[&str] = &[ROOT_ANCHOR_FILENAME, SIGNER_KEY_FILENAME, CHAIN_FILENAME];
```

Comment:
This is me knowing about a couple of more files coming down the pipe and it seemed like the least error prone way to know about the file names in 2 locations.